src/components/blocked-login-error.yml

```diff
@@ -1,9 +1,9 @@
 blockedLoginError:
   loginFailedTooOften:
-    en: Too many failed login attempts within a short time period. Please try again later.
+    en: Too many failed log in attempts within a short time period. Please try again later.
```

Comment:
Actually (I also checked this on LEO), "login" was correct here, as it is a noun here (noun: "the login", verb: "to log in").

Reply:
Indeed, it would have needed to be "Too many failed logging in attempts" 🤔 Corrected!